<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Battlefield</class>
 <widget class="QDialog" name="Battlefield">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>10</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Bomb_lcdNumber">
   <property name="geometry">
    <rect>
     <x>743</x>
     <y>12</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="OpenButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>открыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NewGameButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Новая игра</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>80</y>
     <width>1071</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
